WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Multi-package/SinglePackageHazmatDeclaration.swift

```diff
@@ -113,13 +131,22 @@ private extension SinglePackageHazmatDeclaration {
                                                                       "ship in separate packages.",
                                                                       comment: "Instructions for hazardous package shipping")
         static let hazmatInstructionsSecondSection = NSLocalizedString("Learn how to securely package, label, and ship HAZMAT through " +
-                                                                       "USPS® at www.usps.com/hazmat.",
-                                                                       comment: "Instructions for hazardous package shipping")
-        static let hazmatInstructionsThirdSection = NSLocalizedString("Determine your product's mailability using the USPS HAZMAT Search Tool.",
-                                                                      comment: "Instructions for hazardous package shipping")
+                                                                       "USPS® at %1$@.",
+                                                                       comment: "Instructions for hazardous package shipping. The %1$@ is a tappable link" +
+                                                                       "that will direct the user to a website")
+        static let hazmatInstructionsThirdSection = NSLocalizedString("Determine your product's mailability using the %1$@.",
+                                                                      comment: "Instructions for hazardous package shipping. The %1$@ is a tappable link" +
+                                                                      "that will direct the user to a website")
```

Comment:
super nit: a space between the text in separate lines for translators (similar for other comments below)
```suggestion
                                                                      comment: "Instructions for hazardous package shipping. The %1$@ is a tappable link " +
                                                                      "that will direct the user to a website")
```

Reply:
Indeed! Thanks 😄